src/main/java/anticope/rejects/modules/Boost.java

```diff
@@ -15,24 +15,57 @@ public class Boost extends Module {
     private final Setting<Double> strength = sgGeneral.add(new DoubleSetting.Builder()
         .name("strength")
         .description("Strength to yeet you with.")
-        .defaultValue(4.0)
-        .min(0.5)
+        .defaultValue(0.5)
+        .min(0.1)
         .sliderMax(10)
         .build()
     );
+    
+    private final Setting<Boolean> autoBoost = sgGeneral.add(new BoolSetting.Builder()
+        .name("auto-boost")
+        .description("Automatically boosts you.")
+        .defaultValue(false)
+        .build()
+    );
+    
+    private final Setting<Integer> delay = sgGeneral.add(new IntSetting.Builder()
```

Comment:
Could you change `delay -> delaySetting` and `_delay -> delay`? Its more consistent with java style guide.

```suggestion
    private final Setting<Integer> delaySetting = sgGeneral.add(new IntSetting.Builder()
```